Projects/Shared/DesignSystem/Sources/UIComponents/TodoList/TodoCheckboxComponent.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  TodoCheckboxComponent.swift
+//  Shared
+//
+//  Created by 이지훈 on 6/22/25.
+//  Copyright © 2025 com.noweekend. All rights reserved.
+//
+
+import SwiftUI
+
+struct TodoCheckboxComponent: View {
+    let isCompleted: Bool
+    let title: String
+    let category: String?
+    let categoryColor: Color?
+    let time: String?
+    let onToggle: () -> Void
+    
+    var body: some View {
+        HStack(spacing: 12) {
+            Button(action: onToggle) {
+                if isCompleted {
+                    DS.Images.icnCheckbox
+                        .resizable()
+                        .frame(width: 18, height: 18)
+                } else {
+                    DS.Images.icTodo
+                        .resizable()
+                        .frame(width: 18, height: 18)
+                }
```

Comment:
if else보다 3항연산자도 좋을수있어서 제안해드려용
```suggestion
                    (isCompleted ? DS.Images.icnCheckbox : DS.Images.icTodo)
                        .resizable()
                        .frame(width: 18, height: 18)
```

Reply:
<img width="294" alt="스크린샷 2025-06-22 오후 7 16 43" src="https://github.com/user-attachments/assets/9f2a3017-343d-46a8-ad8a-9065ea0018cd" />
우리형의 따봉을 드립니다